.gemini.yml

```diff
@@ -0,0 +1,22 @@
+rootUrl: http://localhost:8080/components/vaadin-avatar/test/visual/
+gridUrl: http://localhost:4444/wd/hub
+screenshotsDir: ./test/visual/screens
+
+system:
+  plugins:
+    polyserve:
+      port: 8080
+    sauce: true
+
+browsers:
+  chrome:
+    desiredCapabilities:
+      browserName: "chrome"
+      version: "66.0"
+      platform: "Windows 10"
+
+  firefox:
+    desiredCapabilities:
+      browserName: "firefox"
+      version: "47.0"
```

Comment:
IIRC we agreed to remove Firefox, as it's quite outdated. So this part can be removed. 

Not a blocker.

Reply:
Updated.